frontend/javascripts/oxalis/view/rendering_utils.ts

```diff
@@ -59,25 +59,28 @@ export function renderToTexture(
   // Don't respect withFarClipping for the TDViewport as we don't do any clipping for
   // nodes there.
   if (withFarClipping && plane !== OrthoViews.TDView) {
-    function adaptCameraToCurrentClippingDistance(
-      camera: THREE.OrthographicCamera | THREE.PerspectiveCamera,
-    ) {
+    function adaptCameraToCurrentClippingDistance<
+      T extends THREE.OrthographicCamera | THREE.PerspectiveCamera,
+    >(camera: T): T {
       const isArbitraryMode = constants.MODES_ARBITRARY.includes(
         state.temporaryConfiguration.viewMode,
       );
-      camera = camera.clone();
+      camera = camera.clone() as T;
+      // The near value is already set in the camera (done in the CameraController/ArbitraryView).
       if (isArbitraryMode) {
-        camera.far = state.userConfiguration.clippingDistanceArbitrary;
+        // The far value has to be set, since in normal rendering the far clipping is
+        // achieved by the data plane which is not rendered during node picking
+        camera.far = ARBITRARY_CAM_DISTANCE;
       } else {
-        // The near value is already set in the camera (done in the CameraController).
         // The far value has to be set, since in normal rendering the far clipping is
         // achieved by offsetting the plane instead of setting the far property.
         camera.far = state.userConfiguration.clippingDistance;
       }
       camera.updateProjectionMatrix();
+      return camera;
```

Comment:
This is the fix for the actual regression. The camera was cloned and modified (in this function that was introduced in a recent PR), but then later in the original function, the old camera was used again.

Reply:
argh, this explains a lot of the weirdness I experienced with the clipping distances :see_no_evil: thanks for fixing this!